test/e2e/app-dir/next-after-app/index.test.ts

```diff
@@ -1,4 +1,5 @@
 /* eslint-env jest */
+/* eslint-disable jest/no-standalone-expect */
```

Comment:
Probably prefer to disable this on a line instead of for the whole file.

Reply:
oh actually this is unnecessary here, leftover from a different thing